src/OrchardCore/OrchardCore.ResourceManagement/TagHelpers/StyleTagHelper.cs

```diff
@@ -10,17 +10,25 @@ namespace OrchardCore.ResourceManagement.TagHelpers
     [HtmlTargetElement("style", Attributes = AtAttributeName)]
     public class StyleTagHelper : TagHelper
     {
+        private const string IdAttributeName = "asp-id";
```

Comment:
I am flat out today, but I will fix this for you later (before merge), so that any attributes are automatically applied, and we don't need these `asp-id` attributes

Reply:
I tweaked this so that when normal html attributes are supplied to the tag helper it renders them (any of them).

so no special `asp-id` required, just `id` `media` or any other you might need.

Reply:
Thank you !